### DISCUSSION TITLE
blog/why-gateway-api-is-the-future-of-ingress-and-mesh/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#313] [Announcements]
# blog/why-gateway-api-is-the-future-of-ingress-and-mesh/

本文介绍 Kubernetes 中的入口网关及 Gateway API，入口网关与服务网格融合的新趋势。

https://jimmysong.io/blog/why-gateway-api-is-the-future-of-ingress-and-mesh/

**Comment by pflong** [comment]
感觉是目前看到关于 Gateway API 条理最清晰，观点最鲜明的文章，对 Ingress 和 Gateway API 的特点比较也写的非常易懂。反复读了很多遍，顶一下。

> 你在 Gateway 中的路由和策略附件会通过 xDS 协议发送给 Envoy 机群

这里应该是 Envoy 集群。

**Reply by rootsongjc** [reply]
感谢指正，已修改。

**Comment by PeiHongbing1223** [comment]
赞！！！
